html/modules/custom/reliefweb_user_posts/src/Form/UserPostsPageFilterForm.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\reliefweb_user_posts\Form;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\reliefweb_moderation\Form\ModerationPageFilterForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\reliefweb_moderation\ModerationServiceInterface;
+
+/**
+ * Content moderation page filter form handler.
+ */
+class UserPostsPageFilterForm extends ModerationPageFilterForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'reliefweb_moderation_page_filter_form';
```

Comment:
```suggestion
    return 'reliefweb_user_posts_page_filter_form';
```

@attiks I think this should be changed.

That would notably allow us to add a template override: `form--reliefweb-user-posts-page-filter-form.html.twig` for Lisa to add her magic.

Reply:
If I change it, the javascript stops working

Reply:
@attiks I think adding `$form['#attributes']['id'] = 'reliefweb-moderation-page-filter-form';` at end of the `::buildForm()` would make the javascript work.

Reply:
Fixed